src/app/features/ledger/utils/stacks-ledger-utils.ts

```diff
@@ -88,8 +88,9 @@ export function signStacksTransactionWithSignature(transaction: string, signatur
 export function useActionCancellableByUser() {
   const { pathname } = useLocation();
   return (
-    pathname.includes(RouteUrls.DeviceBusy) ||
+    pathname.includes(RouteUrls.ConnectLedger) ||
```

Comment:
I made a change here to allow users to close the dialog if they are on:
- `ConnectLedger` 
- `ConnectLedgerError` 

I stopped allowing close if they are on the `DeviceBusy` route as this is when we are loading their keys so I don't think they should be able to close it